baseapp/baseapp.go

```diff
@@ -165,13 +165,34 @@ func (app *BaseApp) LastBlockHeight() int64 {
 }
 
 // initializes the remaining logic from app.cms
-func (app *BaseApp) initFromStore(mainKey sdk.StoreKey) error {
+func (app *BaseApp) initFromMainStore(mainKey *sdk.KVStoreKey) error {
+
```

Comment:
I don't understand why we add new lines at (non-multiline) function start...but that's just me :man_shrugging:

Reply:
Generally I agree but I think it's cleaner to have a newline in this case because it provides a nice visual cue for the comment directly below - typically comments above code "paragraphs" should have a blank line above them